### ISSUE TITLE
Doesnt block when file path is too long.

### ISSUE THREAD

**Issue by psychophat23** [henrypp/simplewall#1189]
EX:
C:\Windows\SystemApps\Microsoft.Windows.Search_cw5n1h2txyewy\Se

- above is a copy and paste when creating a new rule
- found that it was incomplete upon checking
- and it was not blocking application with long paths
- below is correct full path.

C:\Windows\SystemApps\Microsoft.Windows.Search_cw5n1h2txyewy\SearchApp.exe

**Comment by gettysburg** [comment]
The app should already be listed here or in the last tab of the "Create Rule" form:

![image](https://user-images.githubusercontent.com/2256003/161430205-4a9eeea3-2a53-4505-8479-3789c747c361.JPG)
![image](https://user-images.githubusercontent.com/2256003/161430381-036c4131-2502-4f09-810e-63b354e1f266.png)

This is a screenshot from my server installation, so it lacks most of the normal Windows 10 apps if you are wondering.

You could also save your .XML file and edit it manually using an editor such as Notepad++, simplewall uses the MAX_PATH constant internally (if I remember correctly) so you can use paths with a length up to 254 characters, however I don't know if it would recognize the program this way, as it handles apps differently (as can be seen above).

Let me know if this helped you, if it did, please use the "Close with comment" button.

**Comment by psychophat23** [comment]
Yes thank you for confirming and you're correct that worked. Sorry didn't reply back faster, I tried several ways and copied the rules that were there already. I also used that same process in adding/configuring rules based from/using just the PATH. Instead of manually adding/dropping every executable under the folder to the block list. Configuring the rules based from PATH works, unless I have it too redundant now to list and ports that's why I don't notice it.  =)

Yep that was it . . . THANKS !!!
